### DISCUSSION TITLE
Indicate if a field is localized in UI

### DISCUSSION THREAD

**Discussion by denolfe** [payloadcms/payload#3161] [Feature Requests & Ideas]
# Feature Request

Add visual indicator if a field is localized in the UI.

## Detail

Currently, there is no way for an editor to know if a field is localized. This adds to uncertainty and possibly confusion when switching locales. Adding a visual indicator next to a field if it is localized would be a welcome change.

Example POC: https://github.com/payloadcms/payload/pull/3160

<img width="372" alt="CleanShot 2023-08-11 at 11 50 23" src="https://github.com/payloadcms/payload/assets/65888/487032bb-c1b9-414c-9334-b0f1601b01e5">

**Comment by tylandavis** [comment]
Related https://github.com/payloadcms/payload/discussions/1308

**Comment by tylandavis** [comment]
This could be more useful if it shows the locale currently being edited.
